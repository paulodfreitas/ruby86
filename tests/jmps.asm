.module entry_code
.pseg
.org 0
        irmovl $1, %eax
        irmovl $2, %ebx
        jmp l1
        irmovl $2, %eax
l1:
        subl %ebx, %eax
        jle l2
        irmovl $2, %eax
l2:
        halt
.end
