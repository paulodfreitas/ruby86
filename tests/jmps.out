@ 0000
30
81
00
00
00
00
70
11
00
00
00
30
81
01
00
00
00
30
80
01
00
00
00
30
83
02
00
00
00
61
30
71
2a
00
00
00
30
81
02
00
00
00
30
80
01
00
00
00
30
83
02
00
00
00
61
30
72
43
00
00
00
30
81
03
00
00
00
61
33
73
50
00
00
00
30
81
04
00
00
00
30
80
01
00
00
00
30
83
02
00
00
00
61
30
74
69
00
00
00
30
81
05
00
00
00
30
80
01
00
00
00
30
83
02
00
00
00
61
03
75
82
00
00
00
30
81
06
00
00
00
30
80
01
00
00
00
30
83
02
00
00
00
61
03
76
9b
00
00
00
30
81
07
00
00
00
10
@ 009c
